src/families/tron/utils.js

```diff
@@ -0,0 +1,247 @@
+// @flow
+import bs58check from "bs58check";
+import { BigNumber } from "bignumber.js";
+import get from "lodash/get";
+import type { Transaction, TronOperationMode, TrongridTxInfo } from "./types";
+import type { Account, Operation, OperationType } from "../../types";
+
+export const decode58Check = (base58: string) =>
+  Buffer.from(bs58check.decode(base58)).toString("hex");
+
+export const encode58Check = (hex: string) =>
+  bs58check.encode(Buffer.from(hex, "hex"));
+
+// see: https://solidity.readthedocs.io/en/v0.6.1/abi-spec.html#function-selector-and-argument-encoding
+export const abiEncodeTrc20Transfer = (
+  address: string,
+  amount: BigNumber
+): string => {
+  const encodedAddress = address.padStart(64, "0");
+  const hexAmount = amount.toNumber().toString(16); // convert to hexadecimal
+  const encodedAmount = hexAmount.padStart(64, "0");
+  return encodedAddress.concat(encodedAmount);
+};
+
+export const hexToAscii = (hex: string) => {
```

Comment:
`Buffer.from(hex, "hex").toString("ascii")` ? (this is likely to have better perf)